<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mOptionsDialogLayout">
   <item>
    <widget class="QTabWidget" name="mTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mGridTab">
      <attribute name="title">
       <string>Сетка</string>
      </attribute>
      <layout class="QVBoxLayout" name="mGridTabLayout">
       <item>
        <widget class="QGroupBox" name="mGridGroupBox">
         <property name="title">
          <string>Сетка</string>
         </property>
         <layout class="QVBoxLayout" name="mGridGroupBoxLayout">
          <item>
           <widget class="QCheckBox" name="mShowGridCheckBox">
            <property name="text">
             <string>Показывать &amp;сетку</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="mShowDotsLayout">
            <item>
             <spacer name="mShowDotsSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>32</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="mShowDotsCheckBox">
              <property name="text">
               <string>Показывать &amp;точки вместо линий</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mSnapToGridCheckBox">
            <property name="text">
             <string>&amp;Привязывать к сетке</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="mSnapToVisibleLinesLayout">
            <item>
             <spacer name="mSnapToVisibleLinesSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>32</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="mSnapToVisibleLinesCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Привязывать только к &amp;видимым линиям</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="mGridSettingsLayout">
            <item>
             <layout class="QGridLayout" name="mGridSettingsGridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="mGridSpacingLabel">
                <property name="text">
                 <string>&amp;Шаг сетки:</string>
                </property>
                <property name="buddy">
                 <cstring>mGridSpacingSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="mGridSpacingSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>16</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="mMajorLinesIntervalLabel">
                <property name="text">
                 <string>Интервал между &amp;основными линиями:</string>
                </property>
                <property name="buddy">
                 <cstring>mMajorLinesIntervalSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="mMajorLinesIntervalSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>64</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="mGridSettingsSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mGuidesGroupBox">
         <property name="title">
          <string>Направляющие</string>
         </property>
         <layout class="QVBoxLayout" name="mGuidesGroupBoxLayout">
          <item>
           <widget class="QCheckBox" name="mShowGuidesCheckBox">
            <property name="text">
             <string>Показывать &amp;направляющие</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="mSnapToGuidesLayout">
            <item>
             <spacer name="mSnapToGuidesSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>32</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="mSnapToGuidesCheckBox">
              <property name="text">
               <string>П&amp;ривязывать к направляющим</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mEnableSmartGuidesCheckBox">
            <property name="text">
             <string>&amp;Умные направляющие</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mTabWidget</tabstop>
  <tabstop>mShowGridCheckBox</tabstop>
  <tabstop>mShowDotsCheckBox</tabstop>
  <tabstop>mSnapToGridCheckBox</tabstop>
  <tabstop>mSnapToVisibleLinesCheckBox</tabstop>
  <tabstop>mGridSpacingSpinBox</tabstop>
  <tabstop>mMajorLinesIntervalSpinBox</tabstop>
  <tabstop>mShowGuidesCheckBox</tabstop>
  <tabstop>mSnapToGuidesCheckBox</tabstop>
  <tabstop>mEnableSmartGuidesCheckBox</tabstop>
  <tabstop>mButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mShowGridCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mShowDotsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSnapToGridCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mSnapToVisibleLinesCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mShowGuidesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mSnapToGuidesCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
